app/src/main/java/com/pragati/karuna/util/Extensions.kt

```diff
@@ -0,0 +1,60 @@
+package com.pragati.karuna.util
+
+import android.graphics.drawable.Drawable
+import android.view.View
+import android.widget.TextView
+
+fun TextView.setLeftDrawable(drawable: Drawable?) {
+    this.setCompoundDrawablesWithIntrinsicBounds(
+        drawable,
+        null,
+        null,
+        null
+    )
+}
+
+
+fun View.enable() {
+    alpha = 1f
+    this.isEnabled = true
+}
+
+fun View.disable() {
+    alpha = 0.5f
+    this.isEnabled = false
+}
+
+/**
+ * Extension method to provide show keyboard for View.
+ */
+fun View.gone() {
+    if (visibility != View.GONE) {
+        visibility = View.GONE
+    }
+}
+
+/**
+ * Extension method to provide show keyboard for View.
+ */
+fun View.invisible() {
+    if (visibility != View.INVISIBLE) {
+        visibility = View.INVISIBLE
+    }
+}
+
+/**
+ * Extension method to provide show keyboard for View.
+ */
+fun View.visible() {
```

Comment:
Since now we have this extension functions, may be it would be great to change it everywhere. Also, do we need to check first that whether it is not visible then only make it visible. Is there any performance implication?

Reply:
This method doesn't provide that option.. We might have to create another extensions for that like.. `visibleIfAvailable`, etc

Reply:
Correct.. No need to have the visibility check.. I am removing it..

Reply:
Yes, please go ahead and remove it.